script/Common/00_00.txt

```diff
@@ -199,7 +203,7 @@ Truly―――an unbelievably blue sky.
 [sha:d0566ccb25edb138e68792ae91c13e9e3dd45d3b]{
 -- Page 13, Offset 145.
 -- 白い陽射しより、一点の染みもない青色の方が、残酷すぎて目に痛かった。
-More than the white rays of sunlight, it was the harshness of that unblemished blue that pained my eyes.
+More than the white rays of sunlight, it is the harshness of the unblemished blue that pains my eyes.
```

Comment:
prefer `that` personally for the unblemished blue

Reply:
its